.github/workflows/main.yml

```diff
@@ -81,3 +81,62 @@ jobs:
         with:
           fetch-depth: 0
       - uses: ./tools/github-actions/cascading
+  documentation:
```

Comment:
Maybe it would be cleaner and more readable to be in a separated workflow (with the version as input as it ssems to  be the only required information)